specification/appcomplianceautomation/AppComplianceAutomation.Management/resources/EvidenceResource.tsp

```diff
@@ -0,0 +1,122 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "../models/models.tsp";
+import "./ReportResource.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using OpenAPI;
+using Azure.ClientGenerator.Core;
+
+namespace Microsoft.AppComplianceAutomation;
+/**
+ * A class represent an AppComplianceAutomation evidence resource.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+@parentResource(ReportResource)
+@includeInapplicableMetadataInPayload(false)
+@Azure.ResourceManager.Private.armResourceInternal(EvidenceProperties)
+model EvidenceResource extends Azure.ResourceManager.Foundations.ProxyResource {
+  /**
+   * The evidence name.
+   */
+  @key("evidenceName")
+  @segment("evidences")
+  @visibility("read")
+  @pattern("^[a-zA-Z0-9-_.]+$")
+  @path
+  name: string;
+
+  /**
+   * Evidence property.
+   */
+  @extension("x-ms-client-flatten", true)
+  @extension("x-ms-client-name", "properties")
+  properties: EvidenceProperties;
+}
+
+@armResourceOperations
+@tag("AppComplianceAutomation")
+interface Evidence {
+  /**
+   * Get the evidence metadata
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "For backward compatibility"
+  @operationId("Evidences_Get")
+  get is ArmResourceRead<
+    EvidenceResource,
+    {
+      ...Azure.ResourceManager.Foundations.BaseParameters<EvidenceResource>;
+    }
+  >;
+
+  /**
+   * Create or Update an evidence a specified report
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "For backward compatibility"
+  @operationId("Evidences_CreateOrUpdate")
+  createOrUpdate is ArmResourceCreateOrReplaceSync<
+    EvidenceResource,
+    {
+      ...Azure.ResourceManager.Foundations.BaseParameters<EvidenceResource>;
+      ...ExtraParameter;
+    }
+  >;
+
+  /**
+   *  Delete an existent evidence from a specified report
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "For backward compatibility"
+  @operationId("Evidences_Delete")
+  delete is ArmResourceDeleteSync<EvidenceResource>;
+
+  /**
+   * Returns a paginated list of evidences for a specified report.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-operation-id" "For backward compatibility"
+  @operationId("Evidences_ListByReport")
```

Comment:
`@operationId` is for swagger and it is better to keep same with interface of typespec, in specific, please remove `s` for all `operationdId`, otherwise it will cause breaking changes of SDK when SDK generations changes from swagger to Typespec in the future.
nit: the suggestion applies for all operation.